client/src/main.ts

```diff
@@ -1,15 +1,16 @@
 import Vue from 'vue'
 import App from './App.vue'
-import router from './router'
+import { getVueRouter } from '@/router'
 import store from './store'
 import { getConfig } from '@/plugins/getConfig'
-import vuetify from './plugins/vuetify'
+import Vuetify from "vuetify/lib"
 import KeyCloakService from 'sbc-common-components/src/services/keycloak.services'
 
-import 'quill/dist/quill.core.css'
+import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
 import 'material-design-icons-iconfont/dist/material-design-icons.css'
 import '@/sass/main.sass'
 import '@/sass/overrides.sass'
+import 'quill/dist/quill.core.css'
```

Comment:
Are you sure about moving this CSS after the others? It means that it will override any styles declared in the previous files.

Typically, it's a good idea to have OUR overrides last here.

Maybe add a comment that order matters :)

Reply:
haha thats actually news to me! I was curious about this, will re-order these 👍 Thanks!

Reply:
https://github.com/bcgov/bcrs-business-create-ui/blob/716868610ebe2a07c0c55fe34b52652e28a92950/src/main.ts#L18